src/LinuxTracing/ThreadStateManagerTest.cpp

```diff
@@ -64,17 +72,21 @@ TEST(ThreadStateManager, OneThread) {
 
 TEST(ThreadStateManager, NewTask) {
   constexpr pid_t kTid = 42;
+  constexpr pid_t kWasBlockedByTid = 420;
+  constexpr pid_t kWasBlockedByPid = 4200;
   ThreadStateManager manager;
   std::optional<ThreadStateSlice> slice;
 
-  manager.OnNewTask(100, kTid);
+  manager.OnNewTask(100, kTid, kWasBlockedByTid, kWasBlockedByPid);
```

Comment:
I'm not familiar with this part at all. Shouldn't we include the blockerTid and Pid only in SchedWakeUp @dpallotti ?

Reply:
I added a comment about this in another file. It should be that tid/pid represent the thread that created the thread.

Reply:
But doesn't that just be for Unblocked (which is future work) and not for blocked by?